R/geodist.R

```diff
@@ -60,6 +66,8 @@ step_geodist <- function(recipe,
     rlang::abort("`ref_lon` should be a single numeric value.")
```

Comment:
Can you add some logic to determine if existing objects have `is_lat_lon` and, if not, set to FALSE? We don't want to break existing recipes.